agent-os/commands/plan-product/plan-product.md

```
@@ -109,20 +120,21 @@ Generate `agent-os/product/roadmap.md` with an ordered feature checklist:
 
 Do not include any tasks for initializing a new codebase or bootstrapping a new application. Assume the user is already inside the project's codebase and has a bare-bones application initialized.
 
-#### Creating the Roadmap:
+#### Creating the Roadmap
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h4

(MD001, heading-increment)